components/widgets/editor/raster/RasterChartEditor.js

```diff
@@ -143,24 +159,37 @@ class RasterChartEditor extends React.Component {
     const { loading, bands, error, bandStatsInfo, bandStatsInfoLoading } = this.state;
     const { band, mode, showSaveButton, hasGeoInfo } = this.props;
 
+    let description = band && band.description;
+    const longDescription = description && description.length > 250;
```

Comment:
I think that it's better to use https://lodash.com/docs/4.17.4#truncate. As long as it will truncate test by words => see this example https://stackoverflow.com/a/38150679/3603884